packages/web-app-files/src/router/deprecated.ts

```diff
@@ -0,0 +1,89 @@
+import { RouteConfig, Route, Location, RouteMeta } from 'vue-router'
+import { createLocationSpaces } from './spaces'
+import { createLocationShares } from './shares'
+import { createLocationCommon } from './common'
+
+/**
+ * all route configs created by buildRoutes are deprecated,
+ * this helper wraps a route config and warns the user that it will go away and redirect to the new location.
+ *
+ * @param routeConfig
+ */
+const deprecatedRedirect = (routeConfig: {
```

Comment:
Thank you for going this extra mile! Awesome!

Reply:
Sure :)